WordPress/Classes/Networking/RemotePublicizeService.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc public class RemotePublicizeService : NSObject
+{
+    public var connectURL = ""
+    public var detail = ""
+    public var icon = ""
+    public var jetpackSupport = false
+    public var jetpackModuleRequired = "publicize"
```

Comment:
No big deal! but yes, a little bit. If the initial value is always gonna be overwritten by the backend's response, (maybe) we don't really need an initial value, then.